src/main/java/com/spotinst/sdkjava/model/bl/gcp/ElastigroupScaleDownResponseGcp.java

```diff
@@ -0,0 +1,88 @@
+package com.spotinst.sdkjava.model.bl.gcp;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonInclude;
+
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
```

Comment:
All BL class comments should be same as Api classes.

Reply:
is it removing of redundant class similar as api?  If  yes that is done.